airflow/providers/databricks/sensors/databricks_partition.py

```diff
@@ -0,0 +1,167 @@
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+"""This module contains Databricks sensors."""
+
+from __future__ import annotations
+
+import datetime
+from typing import Sequence
+
+from airflow.exceptions import AirflowException
+from airflow.providers.databricks.hooks.databricks_sql import DatabricksSqlHook
+from airflow.providers.databricks.sensors.databricks_sql import DatabricksSqlSensor
+from airflow.providers.databricks.utils.databricks import ParamEscaper
+from airflow.utils.context import Context
+
+
+class DatabricksPartitionSensor(DatabricksSqlSensor):
+    """Sensor to detect the existence of partitions in a Delta table.
+
+    :param databricks_conn_id: Reference to :ref:`Databricks
+        connection id<howto/connection:databricks>` (templated).
+    :param http_path: Optional string specifying HTTP path of Databricks SQL Endpoint or cluster.
+        If not specified, it should be either specified in the Databricks connection's extra parameters,
+        or ``sql_endpoint_name`` must be specified.
+    :param sql_endpoint_name: Optional name of Databricks SQL Endpoint.
+        If not specified, ``http_path`` must be provided as described above.
+    :param session_configuration: An optional dictionary of Spark session parameters. If not specified,
+        it could be specified in the Databricks connection's extra parameters.
+    :param http_headers: An optional list of (k, v) pairs that will be set
+        as HTTP headers on every request. (templated)
+    :param _catalog: An optional initial catalog to use. Requires DBR version 9.0+ (templated)
+    :param _schema: An optional initial schema to use. Requires DBR version 9.0+ (templated)
+    :param table_name: Table name to generate the SQL query.
+    :param partitions: Partitions to check, supplied via a dict.
+    :param handler: Handler for DbApiHook.run() to return results, defaults to fetch_one_handler
+    :param client_parameters: Additional parameters internal to Databricks SQL Connector parameters.
+    :param partition_operator: Comparison operator for partitions.
+    """
+
+    template_fields: Sequence[str] = (
+        "databricks_conn_id",
+        "_schema",
+        "http_headers",
+        "_catalog",
+        "table_name",
+        "partitions",
+    )
+
+    def __init__(
+        self,
+        table_name: str,
+        partitions: dict,
+        partition_operator: str = "=",
+        *args,
+        **kwargs,
+    ) -> None:
+        super().__init__(*args, **kwargs)
+        self.caller = "DatabricksPartitionSensor"
+        self.partitions = partitions
+        self.partition_operator = partition_operator
+        self.table_name = table_name
+        self.escaper = ParamEscaper()
+
+    def _get_hook(self) -> DatabricksSqlHook:
```

Comment:
This should be covered by the base class - no need to redefine it here

Reply:
Removed them from both child classes.